docs/release-notes.txt

```diff
@@ -6,6 +6,45 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.13
+^^^^^^^^^^^^^^^
+
+**Release Date:** August 5, 2020
+
+**New Features**
+
+- Add the Determined Model Registry. The Model Registry makes it easy to organize models by providing versioning and labeling tools. See :ref:`organizing-models` to get started.
```

Comment:
question: "organize models" -> "organize checkpoints" maybe?

Reply:
I think of checkpoints as somewhat of an implementation detail, personally -- i.e., _models_ are what the _model registry_ organizes. If we want to clarify further we could talk about "trained models"?